csharp/src/Ice/InputStream.cs

```diff
@@ -1537,7 +1539,7 @@ public int[] readIntSeq(int tag)
     /// <param name="tag">The numeric tag associated with the value.</param>
     /// <param name="isset">True if the optional value is present, false otherwise.</param>
     /// <param name="v">The optional value.</param>
-    public void readIntSeq(int tag, out bool isset, out int[] v)
+    public void readIntSeq(int tag, out bool isset, out int[]? v)
```

Comment:
We should eventually be able to get rid of these this out bools, right?

Reply:
Probably not. We want to keep roughly the same public API as in Ice 3.7.